src/octotree.js

```diff
@@ -11,8 +11,10 @@ $(document).ready(() => {
   }
 
   function createAdapter() {
-    const githubUrls = store.get(STORE.GHEURLS).split(/\n/).concat('github.com')
-    return ~githubUrls.indexOf(location.host)
+    const githubUrls = store.get(STORE.GHEURLS).split(/\n/)
+      .map((url) => url.replace(/(.*?:\/\/[^/]+)(.*)/, '$1'))
+      .concat('https://github.com')
+    return ~githubUrls.indexOf(location.protocol + '//' + location.host)
```

Comment:
Because `githubUrls` is array object so `indexOf` does not work (e.g. `['https://whatever.com'].indexOf('whatever.com')`]